experiment/tracer/static/script.js

```diff
@@ -68,8 +67,9 @@ var controller = {
     return logger.options.request.endpoint + encodeURI(logger.options.request.params);
   },
 
-  setMessage: function(msg) {
-    logger.message = msg;
+  setMessage: function(msg, value) {
+    generateView.toggleElement(generateView.messageHolder, true);
+    logger.message = msg.replace('{value}', value);
```

Comment:
Suggest to use [Tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates). The downside is that it does not support IE.

Reply:
Nice, I need to verify that there are not IE users so I can change it.